slack-base/src/main/java/com/hubspot/slack/client/models/events/SlackEventMessageExtendedBase.java

```diff
@@ -0,0 +1,23 @@
+package com.hubspot.slack.client.models.events;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hubspot.slack.client.methods.interceptor.HasUser;
+import java.util.Optional;
+
+public abstract class SlackEventMessageExtendedBase
```

Comment:
@ykaliukh what does `Extended` mean in the model name?

Reply:
@zmarushchak-hs I changed the entire logic, please, check once more.